### ISSUE TITLE
Create example page for real accessibility usability testing

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#4292]
Scope
---
To enhance overall accessibility usability of PDS components, we need a dedicated page with which we can conduct real user testings with people with disabilities.

This page should have the following specs:
- Static 3-4 pages
- Fully functional UI elements
- Real content which makes sense
- As many as possible PDS components
- Public available URL
- Form validation and submission (fake)

Out of Scope
---
List topics that are not part of this task.

Notes  
---
Page can be build within example projects.

Acceptance criteria
---
- [x] Fully functional small website build with PDS components

Subtasks
---
- [ ] Task
